recipes/CVSS/S2ST/hparams/w2v2_large.yaml

```diff
@@ -0,0 +1,209 @@
+###################################
+# Experiment Parameters and setup #
+###################################
+seed: 1234
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+model_name: !PLACEHOLDER
+output_folder: !ref results/<seed>/<model_name>
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+epochs: 200
+keep_checkpoint_interval: 5
+
+use_wandb: True
+
+logger_opts:
+  initializer: !name:wandb.init
+    entity: jar0d
+    project: s2ut_cvss
+    name: !ref <model_name>
+    reinit: True
+    #yaml_config: Null
+    resume: False
+
+#################################
+# Data files and pre-processing #
+#################################
+audio_folder: /local_disk/apollon/jduret/corpus/CV4/fr/clips_16k
```

Comment:
Add placeholders instead of actual path. Also please explain what the different folders are.